src/cmd/mod.rs

```diff
@@ -6,7 +6,7 @@ use crate::{
     wallet::Wallet,
 };
 pub use helium_api::{
-    models::{transactions::PendingTxnStatus, Hnt, Hst, Usd},
+    models::{transactions::PendingTxnStatus, Hnt, Hst, Iot, Mobile, Token, Usd},
```

Comment:
What does `Token` refer to?

Reply:
`Token` is a generic placeholder for `Hnt`, `Hst`, `Mobile` or `Iot`, all of which are Decimals with scale 8.

The type allows us to parse command line arguments as a generic Decimal and then depending on flags elsewhere in the command line, we can convert the type appropriately.